index.js

```diff
@@ -25,9 +35,8 @@ exports.sheltersByZip = function sheltersByZip(req, res) {
   } else {
     const [lat, lon] = zipCodes[zip];
 
-    apiClient.sheltersByLatLon(lat, lon).then(data => {
-      // TODO: Format this response in a way that makes sense for API.AI
-      // (happening in apiClient right now, but broken)
+    apiClient.sheltersByLatLon(lat, lon, Params).then(data => {
```

Comment:
lat long lang, heh. I think this will be a bit simpler when we use an I18n library.